runtimes/pytorch/ubi8-python-3.8/Pipfile

```diff
@@ -49,6 +49,7 @@ requests = "==2.31.0"
 tornado = "==6.3.3"
 traitlets = "==5.1.1"
 urllib3 = "==1.26.18"
+networkx = "==3.1"
```

Comment:
whoa, what happened here?

Reply:
It should be ping on this version to 3.1, because the 3.2 want't available on py8

Reply:
Can i have your review once again :rofl:
Hope to be the last one!

Reply:
Aha, understood. When you updated Pipfile.lock, you did not run whatever command you used on Python 3.8, but you used a newer Python?